src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -103,10 +106,11 @@ class Submitter(stagingBaseDir: File,
     _ <- isValid(stageBag)
     // EASY-1464 3.3.7 checksums
     _ <- samePayloadManifestEntries(stageBag, draftBag)
-    _ <- setRightsRecursively(stageBag.baseDir.parent)
+    draftDepositDir = stageBag.baseDir.parent
+    _ <- setRightsRecursively(draftDepositDir)
     // EASY-1464 step 3.3.9 Move copy to submit-to area
-    _ = logger.info(s"moving ${stageBag.baseDir.parent} to $submitDir")
-    _ = stageBag.baseDir.parent.moveTo(submitDir)
+    _ = logger.info(s"moving $draftDepositDir to $submitDir")
+    _ = srcProvider.move(draftDepositDir.path, submitDir.path, StandardCopyOption.ATOMIC_MOVE)
```

Comment:
good to see the extraction of `stageBag.baseDir.parent`.
I'm wondering though whether you need to change the `moveTo` to a `srcProvider.move` here. Since atomic moves are possible (which you checked on startup), do you still need to be explicit about it here? Can't you just simply use `draftDepositDir.moveTo(submitDir, overwrite = false)`?

Reply:
better safe than sorry with a double check than rely on configuration checks.

Reply:
**Reminder to future me:**

Look at why we were able to pull this off in [`easy-split-multi-deposit`](https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/628bfe965c1e69658b94dd314de318eb3310ca1a/src/main/scala/nl.knaw.dans.easy.multideposit/actions/MoveDepositToOutputDir.scala#L35), but why we can't seem to do the same here.

Reply:
I've been looking at this tonight, but I can't figure out why I don't specify an **atomic move** in `easy-split-multi-deposit`, but do seem to get the correct behavior. I'll try to take another (final) look at it tomorrow with @janvanmansum.